redaxo/src/addons/install/pages/packages.add.php

```diff
@@ -71,18 +71,28 @@
             </thead>
             <tbody>';
 
+    $latest_release = false;
+
     foreach ($addon['files'] as $fileId => $file) {
         $version = rex_escape($file['version']);
+        $release_label = '';
+
+        if (class_exists(rex_version::class) && !rex_version::isUnstable($version) && false == $latest_release) {
+            $release_label = '<br><span class="label label-success">'.rex_i18n::msg('install_latest_release').'</span>';
+            $latest_release = true;
+        }
+
```

Comment:
müsste es nich so sein?

```suggestion
    foreach ($addon['files'] as $fileId => $file) {
        $version = rex_escape($file['version']);
        
        $release_label = '';
        $latest_release = false;
        if (class_exists(rex_version::class) && !rex_version::isUnstable($version)) {
            $release_label = '<br><span class="label label-success">'.rex_i18n::msg('install_latest_release').'</span>';
            $latest_release = true;
        }

```